api/src/components/publication/service.ts

```diff
@@ -15,191 +15,253 @@ export const isIdInUse = async (id: string) => {
-                            id: true,
-                            type: true,
-                            doi: true,
-                            versions: {
-                                select: {
-                                    title: true,
-                                    publishedDate: true,
-                                    currentStatus: true,
-                                    description: true,
-                                    keywords: true
-                                }
-                            }
+                            title: true,
+                            publishedDate: true,
+                            currentStatus: true,
+                            description: true,
+                            keywords: true
                         }
                     }
                 }
-            },
-            linkedFrom: {
-                where: {
-                    publicationFrom: {
-                        versions: {
-                            some: {
-                                isLatestLiveVersion: true
-                            }
-                        }
+            }
+        }
+    },
+    linkedFrom: {
+        where: {
+            publicationFrom: {
+                versions: {
+                    some: {
+                        isLatestLiveVersion: true
                     }
-                },
+                }
+            }
+        },
+        select: {
+            id: true,
+            versionToId: true,
+            publicationFromId: true,
+            draft: true,
+            publicationFrom: {
                 select: {
                     id: true,
-                    versionToId: true,
-                    publicationFromId: true,
-                    draft: true,
-                    publicationFrom: {
+                    type: true,
+                    doi: true,
+                    versions: {
                         select: {
-                            id: true,
-                            type: true,
-                            doi: true,
-                            versions: {
-                                select: {
-                                    title: true,
-                                    publishedDate: true,
-                                    currentStatus: true,
-                                    description: true,
-                                    keywords: true
-                                }
-                            }
+                            title: true,
+                            publishedDate: true,
+                            currentStatus: true,
+                            description: true,
+                            keywords: true
                         }
                     }
                 }
             }
         }
+    }
+} satisfies Prisma.PublicationInclude;
+
+// Ugly, but extend the default include to include private information.
+const privatePublicationInclude = {
```

Comment:
Not sure what to say to this one 🫨

Reply:
Is this just cloning defaultPublicationInclude?

Reply:
This is now written in a friendlier way.